test/_test-helpers.ts

```diff
@@ -44,7 +44,7 @@ export async function testSpunUpApp(appPath: string, originalDir: string) {
   // run typescript
   let resultTS: string
   try {
-    resultTS = await run(`npm run compile`, runOpts)
+    resultTS = await run(`yarn compile`, runOpts)
```

Comment:
I think generally speaking we should keep `npm run` anywhere that it will work, as it's sort of the "safe" choice.